definitions/private-law/json/AuthorisationCaseField/Notifications/AuthorisationCaseField.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "LiveFrom": "01/05/2024",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "fm5ReminderNotifications",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "citizen",
```

Comment:
we need CRUD only for system update . rest all should have R

Reply:
Done